dotCMS/src/main/java/com/dotcms/rest/api/v1/page/PageResource.java

```diff
@@ -217,6 +219,14 @@ public Response saveLayout(@Context final HttpServletRequest request,
                                @PathParam("pageId") final String pageId,
                                final PageForm form) {
 
+        Logger.debug(this, String.format("Saving layout: pageId -> %s layout-> %s", pageId, form.getLayout()));
+
+        if (LicenseManager.getInstance().isCommunity()) {
+
+            Logger.warn(this, String.format("License required to save layout: pageId -> %s layout-> %s", pageId, form.getLayout()));
+            throw new LicenseRequiredException();
```

Comment:
you should include a message here, 
edit:nvm I just see the message is hardcoded on the constructor

Reply:
I am not pretty sure if the license should be check here, @wezell might have a better answer